lib/ecstatic.js

```diff
@@ -80,6 +84,9 @@ var ecstatic = module.exports = function (dir, options) {
       return status[403](res, next);
     }
 
+    if (req.method === 'OPTIONS') {
+      return res.end()
+    }
```

Comment:
As an aside, a --cors setting might be cool too. I should probably whip something up post-merge.